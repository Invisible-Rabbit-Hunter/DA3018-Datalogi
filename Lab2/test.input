1 2 + =
3 4 - =
6 7 * =
43.4 7 / =
1 2 3 + + =
1 2 + 3 + 5 2 + * 6 / =