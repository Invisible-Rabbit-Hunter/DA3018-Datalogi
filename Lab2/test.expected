3.0
-1.0
42.0
6.2
6.0
7.0
